robo_soccer_platform/lib/robo_soccer_platform_web/live/player/player.ex

```diff
@@ -54,22 +57,41 @@ defmodule RoboSoccerPlatformWeb.Player do
     form = Utils.put_form_errors(socket.assigns.form)
 
     if form["errors"] == [] do
-      socket = assign(socket, form: form)
-
-      path =
-        "/player/steering?" <>
-          URI.encode_query(
-            team: team,
-            username: socket.assigns.form["username"],
-            id: socket.assigns.id
-          )
-
-      {:noreply, push_navigate(socket, to: path)}
-    else
-      {:noreply, assign(socket, form: form)}
+      Endpoint.broadcast_from(self(), @is_game_started, "request", %{id: socket.assigns.id, team: team})
```

Comment:
I believe we need to do a similair thing to this from Steering as well, because when a `/player/steering` page is refreshed the new page didn't ever call 
```elixir
push_event(socket, "game_started", %{})
```
and because of that the JS interval responsible for sending joystick updates was never started.

Reply:
Correct, it is made in https://githttps://github.com/Noarkhh/RoboSoccerPlatform/pull/39/fileshub.com/Noarkhh/RoboSoccerPlatform/pull/39/files and it is made with usage of JS and storing data on client side